### PR TITLE
Add `justfile` for env management and common project commands

### PR DESCRIPTION
**Pull request by d33bs**
This PR adds a `justfile` which may be used through [`just`](https://github.com/casey/just) for performing common project commands. I'm adding this to help account for changes over time and to save time when it comes to documentation + reproducibility of the project. I expect to expand this file with other commands but wanted to start with existing needs first and get early input on the format / usage.

Thanks for any thoughts or feedback you may have!

### CODE REVIEW

File: justfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# show a list of just commands for this project
+default:
+  @just --list
+
+# setup conda envs
+@setup-conda-envs:
+    #!{{hashbang}}
+    conda remove -n jump_sc --all
+    conda remove -n R_jump_sc --all
```
Comment (jenna-tomkinson) [inline justfile]: Why have these remove sections? Why not just only have the install environments?

**Review by jenna-tomkinson — APPROVED** [review]
LGTM! I added one question for you to address but other than that this is a super simple and nice addition!

File: justfile (hunk 1 above)
Comment (d33bs) [reply justfile]: Thanks for this - it's a good question! I've encountered behavior from conda in the past that made it so any changes to the `environment.yml` could potentially corrupt or have erroneous behavior when modifying the existing environments. The removals here were a matter of trying to be safe about this. That said, I might be being overprotective here, and will shift to use something like `conda env update --name my_env --file environment.yml --prune` instead.

**Comment by d33bs** [comment]
Thanks @jenna-tomkinson for your review! I've made a small adjustment to help account for your comment. Merging this in now.

**Pull request merged**